app/controllers/coronavirus_form/need_help_with_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::NeedHelpWithController < ApplicationController
```

Comment:
Humm, I'm thinking if there's a gap between the values and human readable strings here.

Is it worth making this less dynamic and just putting it all in a constant at the top of the file?
like:
```
  QUESTION_GROUPS = {
    "feeling_unsafe": "Feeling unsafe",
    "paying_bills": "Paying bills",
    "getting_food": "Getting food",
    "being_unemployed": "Being unemployed or not having any work",
    "going_in_to_work": "Going in to work",
    "somewhere_to_live": "Having somewhere to live",
    "mental_health": "Mental health and wellbeing",
    "I’m not sure": "I’m not sure",
  }.freeze
```

As far as I can tell, those are the keys coming in from the form, the I'm not sure key is a bit odd, but i think that's because the key is being taken from the locale instead, so is inconsistent with the others?

Reply:
It worked when I was plucking the values from the I18n file.

Reply:
I'm going to see if I can figure it out with a fresher brain.

Reply:
@huwd I think I figured it out. Code is super ugly though.